auth-service/src/main/java/com/auth/authservice/presentationlayer/User/UserTokenPair.java

```diff
@@ -0,0 +1,22 @@
+package com.auth.authservice.presentationlayer.User;
+
+import com.auth.authservice.datalayer.user.User;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+
+/**
+ * Created by IntelliJ IDEA.
+ * <p>
+ * User: @Fube
+ * Date: 2021-10-14
+ * Ticket: feat(AUTH-CPC-388)
+ */
+
+@Data
+@Builder(toBuilder = true)
+@Setter
+public class UserTokenPair {
```

Comment:
I know you explained it last time but what is a Token already?

Reply:
The token shows that you are authenticated, it has the username, and expiration date. It is only given when credentials are matched and is sent with every request made to the API to verify it.